### PR TITLE
jax.numpy.convolve frontend function and test. issue #10499

### PR DESCRIPTION
**Pull request by JPGoodale**
Close https://github.com/unifyai/ivy/issues/10499.

### CODE REVIEW

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @JPGoodale 

Thanks for contributing to ivy,
Everything looks great except that the tests are failing for some backends due to precision error during assertion, would you set some tolerance values `atol` and `rtol` for the test function?

Thanks and happy coding😊

**Comment by JPGoodale** [comment]
> Hey @JPGoodale
> 
> Thanks for contributing to ivy, Everything looks great except that the tests are failing for some backends due to precision error during assertion, would you set some tolerance values `atol` and `rtol` for the test function?
> 
> Thanks and happy coding😊

Thanks for the review, I just added some tolerance values and they're passing all tests my machine. Let me know if I need to fix anything else, thanks!

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py [hunk 1]
```diff
@@ -171,6 +171,43 @@ def test_jax_numpy_arctan2(
...
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        rtol=1e-5,
+        atol=1e-5,
+        on_device=on_device,
+        a=x[0],
+        v=x[1],
+        mode='full',
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:206]: Would you add sampling for `mode` as there are other supported modes 
`full`, `same`, and `valid` will merge provided tests are appassing🙂

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Few changes needed and the PR gets merged

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py (hunk 1 above)
Comment (JPGoodale) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py]: Gotcha. Just added it.

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!
Thanks for contributing😊

**Pull request merged**